src/luger_utils.erl

```diff
@@ -34,8 +34,8 @@ single_line(Msg, false) ->
 single_line(Msg, true) ->
     lists:join(" ", binary:split(iolist_to_binary(Msg), [<<"\n">>, <<" ">>], [global, trim_all])).
 
-message(Msg, SingleLine) ->
-    trunc(2048, single_line(Msg, SingleLine)).
+message(Msg, SingleLine, Len) ->
```

Comment:
Consider renaming `Len` to `MaxLen` as I think it describes the variable a bit better.